frontend/src/components/BestExpertiseSection/BestExpertiseSection.jsx

```diff
@@ -15,17 +16,17 @@ const BestExpertiseSection = () => {
       return <Card title={title} details={details} button={button} key={id} />;
     });
     return (
-      <section
-        className="container text-center mt-24"
-        style={{ marginTop: '6.5625rem' }}
+      <Container
+        classNames="text-center mt-24"
+        margin={CONTAINER_OPTIONS.MARGIN.TOP}
```

Comment:
- please, change `classNames` to be `className` without `s`
- and please, remove `mt-24` class from className and use `CONTAINER_OPTIONS.MARGIN.TOW_SIDES` for margin and `CONTAINER_OPTIONS.PADDING.DEFAULT` for padding